Makefile

```diff
@@ -29,7 +29,8 @@ endif
 
 
 # All .go files that are not auto-generated and should be auto-formatted and linted.
-ALL_SRC = $(shell find . -name '*.go' \
+# .sh files and makefiles are included.
+ALL_SRC = $(shell find . \( -name '*.go' -o -name '*.sh' -o -name 'Makefile*' \) \
```

Comment:
This var is meant for go sources. You can define another one for scripts and pass both to the license fixer script